.github/workflows/ci.yml

```diff
@@ -155,6 +155,71 @@ jobs:
           path: target/${{ matrix.target }}/release/${{ matrix.binary || 'sccache' }}${{ endsWith(matrix.target, '-msvc') && '.exe' || '' }}
           if-no-files-found: error
 
+  coverage:
+    name: coverage ${{ matrix.os }} rust ${{ matrix.rustc || 'stable' }} ${{ matrix.extra_desc }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.allow_failure || false }}
+    timeout-minutes: 15
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-18.04
+            rustc: nightly
+            allow_failure: true
+            extra_args: --features=unstable
+          - os: macOS-10.15
+            rustc: nightly
+# Disable for now Windows as it fails with:
+# found invalid metadata files for crate `vte_generate_state_changes`
```

Comment:
Can you refer (ideally here, or the commit message) to the upstream bug report that's blocking us from enabling this?